app/boards/arm/nrfmicro/Kconfig.defconfig

```diff
@@ -0,0 +1,33 @@
+# Electronut Labs Papyr board configuration
+
+# Copyright (c) 2020 Pete Johanson, Okke Formsma, joric
+# SPDX-License-Identifier: MIT
+
+if BOARD_NRFMICRO
+
+config BOARD
+	default "nrfmicro"
+
+if USB
+
+config USB_NRFX
+	default y
+
+config USB_DEVICE_STACK
+	default y
+
+endif # USB
+
+config BT_CTLR
+	default BT
+
+config ZMK_BLE
+	default y
+
+config ZMK_USB
+	default y
+
+config PINMUX
+	default y
```

Comment:
this default seems not to be honored, and has to be set explicitly in each defconfig file. Suggestions?

Reply:
See my note above about the change to the `if` conditional.